deps/SuiteSparse_wrapper.c

```diff
@@ -7,10 +7,6 @@
 #include <string.h>
 #include <cholmod.h>
```

Comment:
Unrelated to this PR: can't we just get rid of this file?

Reply:
I think we can - but that can be a separate PR.